.travis.yml

```diff
@@ -2,15 +2,15 @@
 # * http://www.objc.io/issue-6/travis-ci.html
 # * https://github.com/supermarin/xcpretty#usage
 
-osx_image: xcode9.2
+osx_image: xcode9.3
 language: swift
 # cache: cocoapods
 # podfile: Example/Podfile
 # before_install:
 # - gem install cocoapods # Since Travis is not always on latest version
 # - pod install --project-directory=Example
 script:
-- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/VIPERBuilder.xcworkspace -scheme VIPERBuilder-Example -sdk iphonesimulator11.2 -destination 'platform=iOS Simulator,name=iPhone X,OS=11.2' ONLY_ACTIVE_ARCH=NO | xcpretty
+- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/VIPERBuilder.xcworkspace -scheme VIPERBuilder-Example -sdk iphonesimulator11.4 -destination 'platform=iOS Simulator,name=iPhone X,OS=11.4' ONLY_ACTIVE_ARCH=NO | xcpretty
```

Comment:
I'm seeing a build failure on travis: `xcodebuild: error: SDK "iphonesimulator11.4" cannot be located.`

I believe this needs to be `OS=11.3` & `iphonesimulator11.2`
https://docs.travis-ci.com/user/reference/osx/#Xcode-9.3

Reply:
Yep! It ran locally but wasn't sure of the travis limitations. Thanks!